Learn-CSS-Blog/index33.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav>
+                <ul class="nav-links">
+                    <li><a href="#">Intro</a></li>
+                    <li><a href="#">Blog</a></li>
+                </ul>
+                <a href="#" id="logo" >
+                    <img src="https://d33wubrfki0l68.cloudfront.net/e66392bc2941ad564b6956321e778f29073cf585/4190b/img/glyph-alternative.svg" alt="freeCodeCamp logo">
+                </a>
+            </nav>
+        </header>
+        <section class="container">
+            <div class="about">
+                <div>
+                    <img class="img" src="img/profile.jpg" alt="profile picture">
+                </div>
+                <div class="about-content">
+                    <p>Hello World!</p>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet 
+                        fermentum tortor, at aliquet ante varius nec. Sed vehicula libero sed 
+                        ligula varius, sit amet eleifend velit tincidunt. Fusce eu justo quis 
+                        neque sagittis venenatis. Vivamus ac elit ultricies, laoreet leo vitae, 
+                        rutrum eros. Mauris malesuada auctor quam et facilisis. Nullam ipsum massa,
+                        ultricies id faucibus non, dapibus eu arcu. Fusce non interdum risus.
+                        Ut at lacus mattis, aliquam tellus at, luctus magna. Quisque euismod, 
+                        eros non dictum tincidunt, nisl mauris elementum nibh, vitae facilisis 
+                        leo nulla ut orci. Aenean ut odio eu nisl consectetur facilisis at quis 
+                        metus. Suspendisse potenti. Aenean at varius ex. Interdum et malesuada 
+                        fames ac ante ipsum primis in faucibus. Aliquam ut nisl metus. Nam 
+                        bibendum turpis et vestibulum semper
+                    </p>
+                </div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Copy the second `p` element you just created and paste another right below.
```

Comment:
Copy the second `<p>` element you just created, along with all the text, and paste another right below.